src/tracks/ui/TrackView.cpp

```diff
@@ -161,6 +176,11 @@ int TrackView::GetHeight() const
    return mHeight;
 }
 
+void TrackView::SetTop(int y)
+{
+    DoSetY(y);
+}
```

Comment:
This function does the same thing as the existing SetY().  Should there be only one function?

Reply:
Adding Set/GetTop was not neccessary, actually. I've added them to be used in pair with GetBottom. Using alias names is  common practice i suppose

Reply:
Yes, I added that name at 66e32ca, didn't I now.  I'm criticizing myself.